infra/k8s/redis-values.yaml

```diff
@@ -257,8 +257,8 @@ master:
   ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
   resources:
     requests:
-      memory: 2048Mi
-      cpu: 2000m
+      memory: 14000Mi
```

Comment:
Note that if the user starts an instance type that doesn't have such resources, Redis might not have a place to be scheduled at all. This is currently working fine with `c5.2xlarge` (our default instances), but if the user changes these, they might need to adjust Redis config as well.